Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -1,25 +1,50 @@
+  BTWK_frontier_event.0202_desc_header_vassal:0 "Our liege, [GetScope(destroyed_hadrians_wall_char).Char.GetName], has ordered the demolition of Hadrian's Wall."
+  BTWK_frontier_event.0202_desc_main:0 "\n\n [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] hopes that now the wall no longer serves any meaningful purpose, its stone could be repurposed for the developing settlements throughout [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] realm."
+  BTWK_frontier_event.0202_desc_imperial_like_destroyer:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetFirstName] has done this despite [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] his heritage, from which is intrinsically linked to the momument. For this act, many of his kinsmen have been severely angered."
+  BTWK_frontier_event.0202_desc_else_destroyer:0 "Having no ancient ties to the land, and the wall which sits atop it, [GetScope(destroyed_hadrians_wall_char).Char.GetFirstNamePossessiveNoTooltipRegnal] decision was  tenable. The act has angered the likes of the Britons and of the Latins, who have ancient ties to the monument."
+  BTWK_frontier_event.0202.a_imperial_like:0 "A terrible day for Britannia"
+  BTWK_frontier_event.0202.a_other_0 "Interesting news" 
+  BTWK_frontier_event.0202.a_other_vassal:0 "For the good of the realm!"
+
+  BTWK_frontier_event.0203:0 "Hadrian's Wall Restored"  # ROOT of restoration
+  BTWK_frontier_event.0203_desc:0 "The conclusion of the renovations in [GetGlobalVar('last_old_hadrians_wall_province').MakeScope.Province.GetProvince] has marked the complete restoration of Hadrian's Wall. While the gesture was ultimately symbolic as a reminder of the Imperial roots all Britons share, the wall is now operable as the defensive frontline as it once was, should the need ever arise again. \n\nThough the effort to rebuild has taken many years, gold and manpower to complete, as the overseer of the final stones being set into place, you are the one the poems shall remember the most. Many among the isle are beginning to praise you as 'Hadrianus', as a successor to the great Emperor Hadrian himself. Perhaps this new reputation might aid your bids to become High King of Britannia, should you wish it..."
+  BTWK_frontier_event.0203.a:0 "OPTION"  # Not sure on this
+
+  BTWK_frontier_event.0204:0 "Hadrian's Wall Restored"  # Every other player 
+  BTWK_frontier_event.0204_desc:0 "[GetScope('restored_hadrians_wall_top_liege_char').Char.GetName)] has seen to the final parts Hadrian's Wall's restoration, returning dignity to the land in the wake of Rome's desertion. Many throughout Britannia have begun to praise him as '#italic Hadrianus#!', the heir to the great Roman Emporer Hadrian. Some even say that the completion of such a momentous feat gives [GetScope('restored_hadrians_wall_top_liege_char').Char.GetHerHim)] claim to Britannia's High Kingship... \n\n"
```

Comment:
has seen to the final parts Hadrian's Wall's restoration -> has completed the restoration of Hadrian's Wall
returning dignity to the land in the wake of Rome's desertion -> returning dignity to the land in the wake of the Imperial withdrawal
Emporer -> Emperor
Some even say that the completion of such a momentous feat gives [GetScope('restored_hadrians_wall_top_liege_char').Char.GetHerHim)] claim to Britannia's High Kingship -> Some even say that completing such a momentous feat would make one worthy of the High Kingship... (again needs to be conditional as above)